src/ASM/LR/ASMu2D.C

```diff
@@ -2199,3 +2199,102 @@ bool ASMu2D::evaluate (const FunctionBase* func, RealArray& vec,
   vec = ctrlPvals;
   return ok;
 }
+
+
+ASMu2D::InterfaceChecker::InterfaceChecker(const ASMu2D& pch) :
+  myPatch(pch)
```

Comment:
no line break: `ASMu2D::InterfaceChecker::InterfaceChecker(const ASMu2D& pch) : myPatch(pch)`

Reply:
inconsistent will all other code in this class but changed.